serde/src/test/kotlin/com/amazon/ionhiveserde/configuration/CaseInsensitiveDecoratorTest.kt

```diff
@@ -224,4 +244,24 @@ class CaseInsensitiveDecoratorTest {
         assertSequenceWrapper(sublist[1])
         assertStructWrapper(sublist[2])
     }
+
+    @Test
+    fun ionSequenceCaseInsensitiveDecoratorIteratorNext() {
+        val sequence = sequence_for("[1, '2']")
+        val iter = sequence.iterator()
+        val foo = iter.next()
+        assertEquals(foo, ION.newInt(1))
+        val bar = iter.next()
+        assertEquals(bar, ION.newSymbol("2"))
```

Comment:
```suggestion
        assertEquals(iter.next(), ION.newInt(1))
        assertEquals(iter.next(), ION.newSymbol("2"))
```

`foo` and `bar` aren't really adding anything here.

Reply:
Addressed.